src/ParseServer.js

```diff
@@ -181,9 +181,10 @@ class ParseServer {
     // We pass the options and the base class for the adatper,
     // Note that passing an instance would work too
     const pushController = new PushController();
-
-    const hasPushSupport = pushAdapter && push;
-    const hasPushScheduledSupport = pushAdapter && push && scheduledPush;
+    const hasPushSupport = (pushAdapter && push) != undefined;
+    const hasPushScheduledSupport = hasPushSupport && (scheduledPush == true);
```

Comment:
whoa.  i just tested `undefined && undefined` in the node repl.  never would have guessed that.  nice.